content/get-started/index.md

```diff
@@ -0,0 +1,78 @@
+|How to create a new tutorial in the GTN   |   |[<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial/tutorial.html){:target="_blank"}   |[<i class="fas fa-video" aria-hidden="true"></i>](https://www.youtube.com/watch?v=Vw9h5kvRH2k){:target="_blank"}  |   |   |
+
+### Useful links
+- [FAQs quickstart](https://training.galaxyproject.org/training-material/faq)
+  
+  Feel like you might have a question that many others have had as well? Check our Frequently Asked Questions quickstart!
+
+- [Public Galaxy Servers](https://galaxyproject.org/use/)
+  
+  List of all Public Galaxy Servers available to use or deploy your own version.
+
+- [Galaxy Training Network](https://training.galaxyproject.org/)
+  
+  Collection of tutorials from a variety of topics() developed and maintained by the worldwide Galaxy community.
+
+- [Galaxy Help](https://help.galaxyproject.org/)
+  
+  Have a bug or an error you haven't been able to solve? Do you have a question that you haven't found in already answered in the FAQs? Reach out to the Galaxy Help Community.
+
+- [Galaxy Communities](https://galaxyproject.org/community/)
+  
+  List and description of already stablished communities.
+
+- [Galaxy Publication Library](https://www.zotero.org/groups/1732893/galaxy)
+  
+  Publication library for the Galaxy Project that use, reference, implement, and extend Galaxy.
+
+- [Events](https://galaxyproject.org/events/)
+  
+  Upcoming and past events with Galaxy-related content.
+
+- [Mailing lists](https://galaxyproject.org/mailing-lists/)
+  
+  Links, email address and archives for Galaxy's Mailing Lists, you can reach out to the community in this format, ask your questions and get to know what's up with Galaxy.
+
+- [Working groups](https://galaxyproject.org/community/wg/)
+  
+  List of current Working Groups set within the Galaxy Ecosystem. 
+
+- [Galaxy's repositories](https://github.com/galaxyproject)
+  
+  Galaxy Project Repositories including the GTN,Planemo, Galaxy and The Hub.
+
+### Want to contribute?
+[Here's](https://galaxyproject.org/community/contributing/) our guide on how to start contributing and there are many ways to do so: coding,writing and creating tutorials, answering questions in Gitter or the Help Forum and many more. 
+
+### Bug reporting etiquette
+We have the Help Forum to ask away your questions. If you still are unsure, you can always go to our open issues in GitHub to see if someone reported it, reach out to the Community via Mailing Lists or [create an issue](https://docs.github.com/es/issues/tracking-your-work-with-issues/creating-an-issue) yourself! 
```

Comment:
Could be close to the help forum. We need to go through this information before asking for help in the Galaxy help forum.

Reply:
If useful links is kept, it'd should be on top of it then

Reply:
I would not keep "useful links". The goal of this starter-kit is to guide a newcomer step by step through resources that are most useful when you need them.  

Once you have been through a few tutorials, visite the glossary (and you probably come back very often to check definitions), understood you can use main Galaxy servers for starting your Galaxy journey, understood the Galaxy community (and its regional and scientific communities) and understood that you have you are welcome and have a place (with eventually the possibilty to create your own new community), then it is time to take advantage of the Galaxy website itself. So maybe as a last step, I would briefly explain how the Galaxy website is organized to help a newbee to navigate this jungle. all the resources mentioned in these useful links are easily reachable from the website menus.